src/uct/ib/rc/base/rc_iface.c

```diff
@@ -657,6 +678,8 @@ UCS_CLASS_INIT_FUNC(uct_rc_iface_t, uct_rc_iface_ops_t *ops, uct_md_h md,
 err_destroy_stats:
     UCS_STATS_NODE_FREE(self->stats);
     uct_rc_iface_tx_ops_cleanup(self);
+err_destroy_fence_mp:
```

Comment:
need to have a separate `label` for ` uct_rc_iface_tx_ops_cleanup` as well. 
The issue was here before, please fix, since you are already changing this code